internal/commands/root_test.go

```diff
@@ -139,3 +140,83 @@ func assertError(t *testing.T, err error, expectedMessage string) {
 	assert.Assert(t, err != nil)
 	assert.Assert(t, strings.Contains(strings.ToLower(err.Error()), strings.ToLower(expectedMessage)))
 }
+func Test_validateExtraFiltersForStateExcludeNotExploitableAndOther(t *testing.T) {
+	type args struct {
+		filterKeyVal []string
+	}
+	tests := []struct {
+		filterName      string
+		extraFilterName args
+		replaceValue    []string
+		expectedLog     string
+	}{
+		{
+			filterName:      "Test exclude-not-exploitable",
+			extraFilterName: args{filterKeyVal: []string{"state", "TO_VERIFY;PROPOSED_NOT_EXPLOITABLE;CONFIRMED;URGENT;exclude_not_exploitable"}},
+			replaceValue:    []string{"state", "TO_VERIFY;PROPOSED_NOT_EXPLOITABLE;CONFIRMED;URGENT"},
+			expectedLog:     "",
+		},
+	}
+	for _, test := range tests {
```

Comment:
why do you use this kind of test if you use only one test for each one?
you can use the regular tests...